miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/exception/ValidationError.java

```diff
@@ -0,0 +1,52 @@
+package uk.ac.bbsrc.tgac.miso.service.exception;
+
+import java.io.Serializable;
+
+public class ValidationError implements Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  private static final String generalProperty = "GENERAL";
+  private static final String defaultPropertyMessage = "This value is invalid";
+  private static final String defaultGeneralMessage = "This data is invalid";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)